test/integration/connection_lost_test.exs

```diff
@@ -11,15 +11,37 @@ defmodule MongoosePush.API.ConnectionTest do
     :ok
   end
 
-  # We have to skip it until MIM-748 is resolved
-  @tag :skip
   test "When connection to FCM is lost and regained a PN succeeds" do
     Tools.mock_fcm("/connection", %{"https" => false})
 
-    eventually(assert {500, ""} == Tools.post_conn_error(@url, Tools.sample_notification(:fcm)))
+    # assert :not_ok == try_post(@url, Tools.sample_notification(:fcm), 5)
+    Enum.each(1..5, fn _ ->
+      eventually(assert {500, ""} == Tools.post_conn_error(@url, Tools.sample_notification(:fcm)))
```

Comment:
As I've mentioned - this should be 503 error.

Reply:
It seems there is still code 500 returned

Reply:
So we need to change that :)

Reply:
The reason for that is probably because Sparrow returns unknown error - current MP version does not include the Sparrow version that unlinks the worker

Reply:
It should - this PR should enable the test, update sparrow and bump the version.